frontend/__tests__/src/components/Metrics/MetricsStep/MetricsStep.test.tsx

```diff
@@ -148,9 +148,14 @@ describe('MetricsStep', () => {
       expect(realDoneSettingSection).not.toHaveTextContent(SELECT_CONSIDER_AS_DONE_MESSAGE)
       const columnsArray = within(cycleTimeSettingsSection).getAllByRole('button', { name: LIST_OPEN })
 
-      await userEvent.click(columnsArray[1])
-      const options = within(getByRole('listbox')).getAllByRole('option')
-      await userEvent.click(options[1])
+      await act(async () => {
+        await userEvent.click(columnsArray[1])
+      })
+
+      await act(async () => {
+        const options = within(getByRole('listbox')).getAllByRole('option')
```

Comment:
It should be possible to replace the above `await act` with `const options = await xxx.findAllByRole('option')`